fabtests/common/hmem_ze.c

```diff
@@ -43,7 +43,7 @@
 #include <dlfcn.h>
 #include <level_zero/ze_api.h>
 
-#define ZE_MAX_DEVICES 4
+#define ZE_MAX_DEVICES 8
```

Comment:
Why do we have a hard-coded limit anyway?

Reply:
I did intend to remove the hard-coded limit, but wanted to get a fast fix first. Guess it's better to get it all done now :)